src/mame/misc/astrcorp.cpp

```diff
@@ -970,14 +1368,14 @@ Skill Drop Georgia
 
 No specific PCB model or number....
 
- Astro V02 0022 160pin PQFP ("ASTRO02" silkscreened under chip)
- JX-1689F1028N GRC586.V5 (68K core, has direct connection to program roms)
- Lattice IspLSI 1016 60LJ socketed FPGA
- OKI 6295 clone chip (AD-65 or U6295)
+Astro V02 0022 160pin PQFP ("ASTRO02" silk-screened under chip)
```

Comment:
“Silkscreen” is a single word in English – it doesn’t need a hyphen.

Reply:
Reverted to silkscreen then.